x/cosmwasmpool/keeper.go

```diff
@@ -0,0 +1,45 @@
+package cosmwasmpool
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/osmosis-labs/osmosis/v15/x/cosmwasmpool/types"
+
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+)
+
+type Keeper struct {
+	storeKey   sdk.StoreKey
+	paramSpace paramtypes.Subspace
+
+	// keepers
+	// TODO: remove nolint once added.
+	// nolint: unused
+	poolmanagerKeeper types.PoolManagerKeeper
+	// TODO: remove nolint once added.
+	// nolint: unused
+	contractKeeper types.ContractKeeper
+	// TODO: remove nolint once added.
+	// nolint: unused
+	wasmKeeper types.WasmKeeper
```

Comment:
Note to reviewer: TODOs are tracked in:

Reply:
Yes, I'm defining the bare minimum of what interface is required by this module in `types.expected_keepers.go`